plugins/destination/postgresql/client/types.go

```diff
@@ -106,76 +106,73 @@ func (c *Client) PgToSchemaType(t string) schema.ValueType {
 	}
 }
 
-func (*Client) Pg10ToSchemaType(t string) schema.ValueType {
+func (c *Client) Pg10ToSchemaType(t string) arrow.DataType {
 	if strings.HasPrefix(t, "timestamp") {
-		return schema.TypeTimestamp
+		return arrow.FixedWidthTypes.Timestamp_us
```

Comment:
Maybe add a comment that PostgresSQL permits up to microseconds (6 digits) precision, as for the reason for choosing micros seconds here

Reply:
Added - https://www.postgresql.org/docs/current/datatype-datetime.html